### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by nheyek**

### CODE REVIEW

File: app/src/androidTest/java/ch/epfl/sweng/studyup/LoginActivityTest.java [hunk 1]
```diff
@@ -36,7 +36,7 @@ public void disableMock() {
         Utils.isMockEnabled = false;
     }
 
-    // Test handle failed login message
+    // Test handle failed login message}
```
Comment (Amustache) [inline app/src/androidTest/java/ch/epfl/sweng/studyup/LoginActivityTest.java]: Typo? haha

Comment (nheyek) [reply app/src/androidTest/java/ch/epfl/sweng/studyup/LoginActivityTest.java]: Hahah yes you're right.

File: app/src/androidTest/java/ch/epfl/sweng/studyup/CustomActivityTest.java [hunk 2]
```diff
@@ -1,96 +1,52 @@
...
 
-    @Test
-    public void Z_checkDisplayAndAccessToGallery() throws Exception {
         onView(withId(R.id.pic_btn)).perform(click());
-        assertTrue(device.findObject(new UiSelector().text(Utils.CAMERA)).exists());
-        assertTrue(device.findObject(new UiSelector().text(Utils.GALLERY)).exists());
-        assertTrue(device.findObject(new UiSelector().text(Utils.CANCEL)).exists());
-        clickButton(Utils.GALLERY);
-        clickButton(Utils.JUSTONCE);
-    }
```
Comment (leobouraux) [inline app/src/androidTest/java/ch/epfl/sweng/studyup/CustomActivityTest.java]: Why did you delete all this ? They didnt work ??
